scalding-parquet/src/test/scala/com/twitter/scalding/parquet/thrift/PartitionedParquetThriftSourceTests.scala

```diff
@@ -0,0 +1,60 @@
+package com.twitter.scalding.parquet.thrift
+
+import java.io.File
+
+import com.twitter.scalding._
+import com.twitter.scalding.parquet.thrift_java.test.Address
+import org.apache.hadoop.fs.Path
+import org.apache.parquet.hadoop.ParquetReader
+import org.apache.parquet.thrift.ThriftParquetReader
+
+import org.scalatest.{ Matchers, WordSpec }
+
+object PartitionedParquetThriftTestSources {
+  val path = "/a/path"
+  val partitionSource = PartitionedParquetThriftSource[String, Address](path, "%s")
+}
+
+class PartitionedParquetThriftWriteJob(args: Args) extends Job(args) {
+  import PartitionedParquetThriftTestSources._
+  val input = Seq(new Address("123 Embarcadero", "94111"), new Address("123 E 79th St", "10075"), new Address("456 W 80th St", "10075"))
+
+  TypedPipe.from(input)
+    .map { address => (address.getZip, address) }
+    .write(partitionSource)
+}
+
+class PartitionedParquetThriftSourceTests extends WordSpec with Matchers {
+  import PartitionedParquetThriftTestSources._
+
+  def validate(path: Path, expectedAddresses: Address*) = {
+    val parquetReader: ParquetReader[Address] =
+      ThriftParquetReader.build(path).withThriftClass(classOf[Address]).build()
```

Comment:
cool, can you update the docs to explain the requirements on P being a String or tuple of Strings? I guess that might not be true if you provide a different TupleSetter, but at least taht part should be clarified too (that P and the TupleSetter need to match, which is not tracked by the type system)